src/Employer/Employer.Web/Views/Part1/Training/TrainingHelp.cshtml

```diff
@@ -0,0 +1,24 @@
+﻿@using Microsoft.Extensions.Options
+@inject IOptions<ExternalLinksConfiguration> externalLinks
+
+<a asp-route="@RouteNames.Training_First_Time_Get" class="govuk-back-link govuk-!-margin-top-0 govuk-!-margin-bottom-6">Back</a>
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <h1 class="govuk-heading-xl">
+            Find apprenticeship training
+        </h1>
+        <p class="govuk-body">You will need to find appropriate training for any apprentices you take on. You will also need to find training providers who offer the apprenticeship training you choose.</p>
+        <p class="govuk-body">You can contact the National Apprenticeship Service for advice or help using the service.</p>
+        
+        <p class="govuk-body">
+            <strong>National Apprenticeship Service</strong><br>
+            Telephone: @RecruitViewConstants.HelpdeskPhoneNumber
+            <a href="@RecruitViewConstants.HelpdeskPhoneCharges">Find out about call charges</a>
+        </p>
+        
+        <a class="govuk-button" href="@externalLinks.Value.FindApprenticeshipTrainingUrl">Find apprenticeship training</a>
+        <a asp-route="@RouteNames.Dashboard_Index_Get" class="govuk-link das-button-link">Cancel</a>
```

Comment:
👍  Prototype had changed mid development!! This will be done in ER-1337 (There is a lot of blur between the stories, my main concern in this ticket is to split the pages and get the routes correct)